src/main/java/seedu/address/model/session/Session.java

```diff
@@ -169,6 +173,18 @@ public Duration getSessionDuration() {
         return Duration.between(this.startDateTime, this.endDateTime);
     }
 
+    @Override
+    public String toString() {
+        final StringBuilder builder = new StringBuilder();
+        builder.append("Type: ").append(type)
+                .append(" Start: ").append(startDateTime.format(FORMAT_DATE_TIME))
+                .append(" End: ").append(endDateTime.format(FORMAT_DATE_TIME))
+                .append(" Module Code: ").append(moduleCode)
+                .append(" Description: ").append(description)
+                .append(" Recurs: ").append(isRecurring);
+        return builder.toString();
+    }
```

Comment:
The string message is actually quite long. Not all of it is visible in the line. Would be a good idea to add some new lines (\n).

Reply:
No problem! I actually have an idea of how to make this part of my user visible feature. Instead of printing a message, it'll show all the info in a ui card in the result box.